librz/core/cmd_api.c

```diff
@@ -79,6 +79,38 @@ RZ_IPI int rz_output_mode_to_char(RzOutputMode mode) {
 	return -1;
 }
 
+RZ_IPI RzOutputMode rz_char_to_output_mode2(char input) {
```

Comment:
why is it needed?

Reply:
Hi. Sorry for the late response. I was using this function find the source of an error. I'll remove it once I find the error.